BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -2628,6 +2667,24 @@ public override string GetInfo()
             output.AppendLine($"Min Range: {minStaticLaunchRange} m");
             output.AppendLine($"Max Range: {maxStaticLaunchRange} m");
 
+            missileType = missileType.ToLower();
+
+            if (missileType == "missile")
+            {
+                if(decoupleBoosters)
+                {
+                    if(boosterMass > 0) dryMass -= boosterMass;
+                    if(sustainerMass > 0) dryMass -= sustainerMass;
+                }
+                dryMass = (float) Math.Round(Convert.ToDouble(dryMass),4);
+                output.AppendLine($"Dry mass: {dryMass} T");
+            }
```

Comment:
I think this is likely just going to confuse people. knowing the dry mass isn't really helpful unless you also know the fuel mass, and SRB geometry, and thrust is dependent upon it, or are trying to calculate missile max speed in their heads on the fly. Swap out with a missile acceleration value?

Reply:
yeah thinking about it now i do think knowing the dry weight can lead to some confusion since to know the fuel mass the person would need to deduce the dry from the total and would got the total fuel mass but don't know how much each stage have, i will think about how i can calculate an acceleration value or a max speed value

Reply:
Now it's total DeltaV the missile have so i think should be more user friendly